docs/CUSTOM_RULES_101.md

```diff
@@ -86,7 +86,7 @@ It's also the property `<sonarQubeMinVersion>` which guarantees the compatibilit
 <plugin>
   <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
   <artifactId>sonar-packaging-maven-plugin</artifactId>
-  <version>1.18.0.372</version>
+  <version>...</version>
```

Comment:
Is the `...`  intentional? Both poms in the template use `1.20.0.405`

Reply:
Indeed. I wanted to change it and forgot.